src/typescript/frontend/src/components/OrderbookTable.tsx

```diff
@@ -69,12 +68,22 @@ const Row: React.FC<{
   );
 };
 
-export function OrderBook({ marketData }: { marketData: ApiMarket }) {
+export function OrderbookTable({
+  marketData,
+  data,
+  isFetching,
+  isLoading,
+}: {
+  marketData: ApiMarket;
+  data: Orderbook | undefined;
+  isFetching: boolean;
+  isLoading: boolean;
+}) {
   const [precision, setPrecision] = useState<Precision>(precisionOptions[0]);
-  const { data, isFetching, isLoading } = useOrderBook(
-    marketData.market_id,
-    precision
-  );
+  // const { data, isFetching, isLoading } = useOrderBook(
+  //   marketData.market_id,
+  //   precision
+  // );
```

Comment:
Can we avoid putting commented out code in here?

Reply:
Removed